promptsource/templates/jigsaw_unintended_bias/templates.yaml

```diff
@@ -0,0 +1,212 @@
+dataset: jigsaw_unintended_bias
+templates:
+  17f730bd-fc7e-419c-84cc-e5c795268d02: !Template
+    answer_choices: null
+    answer_choices_key: null
+    id: 17f730bd-fc7e-419c-84cc-e5c795268d02
+    jinja: 'Online comments can sometimes be toxic. For instance, they might be insulting, threatening, racist or
+    many other issues. If 0 corresponds to a very civil comment, and 100 to a highly toxic one, how would you grade the
+    following comment?
+
+    "{{comment_text}}"
+
+    |||
+
+    {{(target * 100.0) | round(3) }}
+    '
+    metadata: !TemplateMetadata
+      choices_in_prompt: null
+      metrics:
+      - Other
+      original_task: true
+    name: online_comment
+    reference: ''
+  1add6d25-346e-4bf5-aa58-8dc184018f9e: !Template
+    answer_choices:
+    - keep
+    - remove
+    answer_choices_key: null
+    id: 1add6d25-346e-4bf5-aa58-8dc184018f9e
+    jinja: 'You are the moderator of the comments section on a website. You want to remove uncivil comments.
+    Would you {{answer_choices[1]}} or {{answer_choices[0]}} the following comment?
+
+    "{{comment_text}}"
+
+    |||
+
+    {{answer_choices[target | round | int]}}
```

Comment:
same

Reply:
If same refers to conditional, this is not needed as explained above.
If it refers to rounding, in this instance the problem is cast as binary classification so not appropriate.